src/components/PastTripNotes.js

```diff
@@ -1,24 +1,44 @@
 import styled from 'styled-components';
 import { FiTrash } from 'react-icons/fi';
+import Modal from './Modal';
+import { useState } from 'react';
+
+export default function PastTripNotes({ notes, onDelete }) {
+  const [visible, setVisible] = useState(false);
+  const [id, setId] = useState('');
 
-export default function PastTripNotes({ notes, onDeleteNote }) {
   return (
     <ul>
       {notes.map(({ note, _id }) => {
         return (
-          <ListEntries key={_id}>
+          <ListEntry key={_id}>
             <p>{note}</p>
-            <Delete onClick={() => onDeleteNote(_id)}>
+            <Delete onClick={() => handleId(_id)}>
               <FiTrash />
             </Delete>
-          </ListEntries>
+          </ListEntry>
         );
       })}
+      {visible && (
+        <Modal
+          onDelete={() => handleDelete(id)}
+          onKeep={() => setVisible(!visible)}
+        />
+      )}
     </ul>
   );
+
+  function handleId(id) {
```

Comment:
same here with the id. 
But i assume its maybe a different id? 
Well, you get the point either way, if its not the same, just ignore my prior comment on the id as well :D